packages/stacks-docs/_data/menu.json

```diff
@@ -3,22 +3,37 @@
     {
       "class": ".s-menu",
       "applies": "N/A",
-      "description": "Base container styling for a menu"
+      "description": "Base container styling for a menu."
     },
     {
-      "class": ".s-menu--title",
+      "class": ".s-menu--divider",
       "applies": ".s-menu",
-      "description": "Adds appropriate styling for a title within a menu"
+      "description": "Adds a divider line between menu sections."
     },
     {
-      "class": ".s-menu--divider",
+      "class": ".s-menu--item",
```

Comment:
I am unsure about the naming here. Since this can be also a button should we call it something more generic (`s-menu--action`)